### PR TITLE
Add bank custom expiry in bank transfer param and bank transfer param…

### PR DESCRIPTION
**Pull request by adrikacang**
Add cutomexpiry paramater in banktransfer request

### CODE REVIEW

**Comment by adamnoto** [comment]
hi @adrikacang, please move the test file accordingly (should be in `test` folder not in `src`).

**Comment by adrikacang** [comment]
@saveav affirmative

**Comment by adamnoto** [comment]
LGTM, please help review and merge @drabiter

File: src/main/java/id/co/veritrans/mdk/v1/gateway/model/CustomExpiry.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    }
+
+    /**
+     * Custom Expiry constructor
+     *
+     * @param orderTime Time when the order is created in merchant website
+     * @param expiryDuration Time duration the payment will remain valid
+     * @param unit Unit for expiryDuration. Valid values are minute,hour, or day
+     */
+    public CustomExpiry(final Date orderTime, final Integer expiryDuration, final Unit unit) {
```
Comment (drabiter) [inline src/main/java/id/co/veritrans/mdk/v1/gateway/model/CustomExpiry.java:45]: `orderTime` and `unit` are optional (can be not sent on the API).

File: src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ * Testing for BankTransferRequest parameters
+ */
+public class BankTransferParamTest {
+
+    @Test
+    public void testBankTransfer() {
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss Z");
+        Date date = new Date();
+        try {
+            date = sdf.parse("2016-12-07 11:54:12 +0700");
```
Comment (drabiter) [inline src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java]: there should be not `catch` guard unless we want to assert the exception, so `Date date = sdf.parse("2016-12-07 11:54:12 +0700");` should be sufficient.

File: src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+        } catch (ParseException e) {
+            e.printStackTrace();
+        }
+
+        BankTransferRequest param = new BankTransferRequest();
```
Comment (drabiter) [inline src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java]: `param` is less descriptive variable name.

File: src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import static org.testng.Assert.assertTrue;
+
+/**
+ * Testing for BankTransferRequest parameters
+ */
+public class BankTransferParamTest {
```
Comment (drabiter) [inline src/test/java/id/co/veritrans/mdk/v1/integration/vtdirect/BankTransferParamTest.java:20]: `BankTransferRequestTest`

**Review by drabiter — CHANGES_REQUESTED** [review]

**Comment by adamnoto** [comment]
wokee, thanks kak @drabiter. @adrikacang make the necessary changes pls. thankss.

**Comment by adrikacang** [comment]
Okay updated. Thank you for the feedback.

**Review by drabiter — APPROVED** [review]

**Pull request merged**